db/redis.js

```diff
@@ -162,6 +163,34 @@ async function getInnosentWindow(locationID, startTime, endTime, windowLength) {
   return radarStream
 }
 
+async function getInnosentTimeWindow(locationID, windowSize) {
+  const windowSizeinMilliseconds = windowSize * 1000
+  const currentTime = await getCurrentTimeinMilliseconds()
+  let startTime = currentTime - windowSizeinMilliseconds
+
+  if (isTestEnvironment) {
```

Comment:
so both of my new functions have this. I only added this because of weirdness in Travis with testing. Without this, I get the error:

`	Error computing Innosent Moving Average: ReplyError: ERR Invalid stream ID specified as stream command argument
`
for each time this function is called in the tests. Weirdly this did not occur when just running tests locally with npm test (before I added the flag).